pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -192,9 +194,23 @@ var kubeletServerUsages = []capi.KeyUsage{
 }
 
 func isSelfNodeServerCert(csr *capi.CertificateSigningRequest, x509cr *x509.CertificateRequest) bool {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletServerCertificate) {
```

Comment:
ok, then move the gate to recognizers(), and remove the gate-twiddling from the init()